java/se/sics/mspsim/cli/ProfilerCommands.java

```diff
@@ -106,11 +106,11 @@ public int executeCommand(final CommandContext context) {
               namematch = context.getArgument(i++);
               if (i < context.getArgumentCount()) {
                   // Multiple patterns
-                  namematch = "(" + namematch;
-                  for(; i < context.getArgumentCount(); i++) {
-                      namematch += "|" + context.getArgument(i);
+                var namematchBuilder = new StringBuilder("(" + namematch);
```

Comment:
Since a StringBuilder is introduced, why still concatenate strings?

Reply:
Fixed.